packages/carbon-web-components/src/components/toggle-tip/toggletip.scss

```
@@ -10,6 +10,7 @@
 @use '@carbon/styles/scss/config' as *;
 @use '@carbon/styles/scss/type' as *;
 @use '@carbon/styles/scss/utilities';
+@use '@carbon/styles/scss/utilities/custom-property';
```

```suggestion
@use '@carbon/styles/scss/utilities' as *;
```

and then maybe we can remove the `@include custom-property.declaration(` to `@include declaration(`